packages/editor/src/extensions/image/component.tsx

```diff
@@ -116,13 +123,47 @@ export function ImageComponent(
           }
         }}
       >
+        <ResponsivePresenter
+          isOpen={Boolean(resizing)}
+          desktop="popup"
+          mobile="none"
+          blocking
+          focusOnRender={false}
+          onClose={() => {}}
+          position={{
+            target: imageRef.current || "mouse",
+            align: "start",
+            location: "top",
+            yOffset: 2,
+            isTargetAbsolute: true
+          }}
+        >
+          <Box
+            sx={{
+              background: "var(--background-secondary)",
+              px: 2,
+              py: 1,
+              borderRadius: "default"
+            }}
+          >
+            <Text variant="subBody" sx={{ fontWeight: "bold" }}>
+              {resizing?.width}
+              {" × "}
+              {resizing?.height}
+            </Text>
+          </Box>
+        </ResponsivePresenter>
```

Comment:
Why do we need `ResponsivePresenter` instead of simply position the `Box` where we want?